tools/system_libs.py

```diff
@@ -1089,6 +1090,8 @@ def get_files(self):
           'pthread_getschedparam.c',
           'pthread_setschedprio.c',
           'pthread_setconcurrency.c',
+          'default_attr.c',
+          'lock_ptc.c',
```

Comment:
Is this necessary? AFAIK, `__inhibit_ptc()` / `__release_ptc()` is not necessary within non-pthread builds, since `pthread_rwlock_t` is no-op anyway.

Reply:
Needed by `pthread_attr_init.c` above.   However, I do see that perhaps we can just use dummies instead.  Let me see.

Reply:
Fixed